gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -1286,3 +1286,9 @@ L.hud_revival_time = "{time}s"
 
 -- 2020-05-03
 L.door_destructible = "Tür ist zerstörbar ({health}HP)"
+
+--2020-05-28
+L.confirm_detective_only = "Nur Detektive können Leichen bestätigen"
+L.inspect_detective_only = "Nur Detektive können Leichen untersuchen"
+L.corpse_hint_no_inspect = "Nur ein Detektiv kann diesen Körper untersuchen."
+L.corpse_hint_inspect_only = "Drücke [{usekey}] zum Durchsuchen. Nur Detektive können diesen Körper bestätigen."
```

Comment:
is there a reasoning behind these not sticking to either `Leichen` or `Körper` ?

Reply:
corpse and body

Reply:
the english strings are all 'body' tho, that's why i'm asking

Reply:
oh, hm, I dunno. It's always hard to translate it into german and to keep continuity because I always use it in english

Reply:
Seeing as the whole german language strings jumps between `Leichen` and `Körper` it doesn't matter to much for now. i would still prefer it to be only one of the two.